docs/figure_file_format.rst

```diff
@@ -0,0 +1,232 @@
+
+The description of the FileAnnotation can be used to specify an Image to show as a thumbnail
+in the File > Open listing of files::
+
+    {
+        "imageId": 123,
+        // only needed if image comes from a different server
+        "baseUrl": "demo.openmicroscopy.org",
+    }
+
+In general, OMERO.figure FileAnnotations are *not* linked to any Images because this would
+display them in the client UI alongside a delete button, allowing the user to accidentally delete
+the file, particularly since they may not recognize the file.
+
+JSON format
+-----------
+
+The JSON data format consists of a single object that represents the figure and contains a list of ``panels``.
+There are a number of versions of the format, with minor differences between them. If you
+open an older version in the OMERO.figure app, it will handle the update and the file will
+be saved using the latest version.
+However, the figure export script may not handle older versions.
+
+This is an example of a minimal OMERO.figure JSON file::
+
+
+    {
+    // see older versions below
+    "version": 5,
+    "panels": [
+        {
+            // position of the panel on the page
+            "x": 200.0,
+            "y": 200.0,
+            "width": 150,
+            "height": 100,
+            // Z and T index
+            "theZ": 0,
+            "theT": 0,
+            // channel info & rendering settings
+            "channels": [
+                {
+                    "label": "528.0",
+                    "color": "00FF00",
+                    "window": {
+                        "min": 0,
+                        "max": 1542.5,
+                        "start": 0,
+                        "end": 1542.5
+                    },
+                    "active": true
+                }
+            ],
+            // Image metadata
+            "imageId": 1267,
+            "name": "U20S-RCC1.11_R3D_FRAP.dv",
+            "sizeZ": 1,
+            "sizeT": 64,
+            "orig_width": 200,
+            "orig_height": 200,
+        }
+    ],
+    }
+
+
+Optional settings for each panel::
+
+    // pixel size x allows user to add scalebar
+    "pixel_size_x": 0.107,
+    "pixel_size_y": 0.107,          // not used yet
+    "pixel_size_x_symbol": 'µm',    // µm by default
+    // options are omero.model.LengthI.SYMBOLS.keys()
+    "pixel_size_x_unit": 'MICROMETER',
+
+    // show a scalebar
+    "scalebar": {
+        "show": true,
+        "length": 10,
+        "units": "MICROMETER",
+        "position": "bottomright",  // topright, topleft, bottomleft
+        "color": "FFFFFF",
+        "show_label": true,
+        "font_size": "14"
+    },
+
+    // Timestamps in seconds, 1 per T-index. Used for time-stamp labels and T-slider
+    "deltaT": [
+        -0.94, -0.61, -0.28, 0.05, 0.33, 0.65
+    ],
+
+    // labels on the panel
+    // positions are: top, left, right, leftvert, bottom, topleft, topright, bottomleft, bottomright
```

Comment:
outside this PR the option ``rightvert`` is missing to match the ``leftvert`` option